pkgs/desktops/gnome-3/core/dconf-editor/default.nix

```diff
@@ -3,13 +3,13 @@
 
 let
   pname = "dconf-editor";
```

Comment:
```suggestion
```

Reply:
Cleaned all this up. I've been avoiding doing cleanup, since I was told in a PR review it's wrong to do together with updates :woman_shrugging: